src/web/app/components/student-list/student-list.component.ts

```diff
@@ -64,6 +65,12 @@ export class StudentListComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  ngAfterContentChecked(): void {
```

Comment:
We should not change the content of `@Input()` property. You should create a new variable instead.

Reply:
Actually, have you consider the solution where we sort the student in the places where the component is being used?

Reply:
Yes I have thought about that, but that would mean taking the sortBy function out to different places and that would result in code duplication? Not sure what would be the best way to do so.

Reply:
Personally, I would prefer to change the single point of truth. I don't think it will result in considerable duplication.